src/gaen/nativeModule.ts

```diff
@@ -191,6 +192,8 @@ export const detectExposures = async (): Promise<DetectExposuresResponse> => {
         return { kind: "failure", error: "NotEnabled" }
       case "NotAuthorized":
         return { kind: "failure", error: "NotAuthorized" }
+      case "Unknown":
```

Comment:
Can you confirm Unknown is being sent on both Android and iOS when airplane mode or networking is off? @lundjrl

Reply:
@artgibson Android shows "Can't check for exposures" and "Enable bluetooth" dialogs when this test is ran.

Reply:
Thanks sounds good